### DISCUSSION TITLE
Binding Directly To Model Properties on V3

### DISCUSSION THREAD

**Discussion by robsontenorio** [livewire/livewire#8014] [1. Help] [answered]
Is this approach supported on Livewire 3?

https://laravel-livewire.com/docs/2.x/properties#binding-models

Trying like V2 docs says, but on V3 I am getting 

<img width="389" alt="image" src="https://github.com/livewire/livewire/assets/118955/3eaa360d-7e7b-4687-a10b-0af86b0d932b">

**Comment by robsontenorio** [comment]
Just saw it is legacy now.

**Reply by joshhanley** [reply, marked as answer]
@robsontenorio yeah we don't recommend it. We have the legacy option so if people were heavily using it in V2, that they can enable legacy mode to keep everything running when upgrading to V3. But for new development we say not to use it.
